src/osp/UserInputHandler.cpp

```diff
@@ -264,8 +270,9 @@ ButtonControlHandle UserInputHandler::config_get(std::string const& name)
 
     if (cfgIt == m_controlConfigs.end())
     {
-        // Config not found, crashes after this, so needs to be fixed
-        std::cout << "config not found!\n";
+        // Config not found, no way to have an empty key so far, so throw an exception
+        std::cout << "No config for " << name << std::endl;
+        throw std::runtime_error("Error: no config with " + name);
         return ButtonControlHandle(nullptr, 0);
```

Comment:
You don't need a return statement after a throw statement. The return statement will never be reached.